docs/_snippets/cloud/integrations/permissions-and-security.mdx

```diff
@@ -1,5 +1,9 @@
 #### Permissions and security
 
-Elementary cloud doesn't need permissions to your tables and schemas.
-It is recommended to create a read only user for the elementary schema only.
+Elementary cloud doesn't require read permissions to your tables and schemas, but only the following:
+
+- Read-only access to the elementary schema.
+- Access to read metadata in information schema and query history, related to the tables in your dbt project.
+
+It is recommended to create a user using the instructions specified above to avoid granting excess privileges.
```

Comment:
Change from today (was merged to master) - a fix to include that we rely on metadata and not only on the elementary schema